smoke-tests/spec/route53_helper.rb

```diff
@@ -0,0 +1,54 @@
+# Create new Route53 zone
+# Expect a domain name in input
+# Return route53 zone object https://docs.aws.amazon.com/sdkforruby/api/Aws/Route53/Types/CreateHostedZoneResponse.html
+def create_zone(domain)
+  client = Aws::Route53::Client.new
+  client.create_hosted_zone(
+    name: domain, # required
```

Comment:
I don't think you need these `# required` comments. 

If you try to call `create_zone` without supplying a parameter, you'll get a runtime error